apps/multisig/src/layouts/Settings/index.tsx

```diff
@@ -65,7 +66,14 @@ const Settings = () => {
         </div>
         <div className="grid gap-[32px] grid-cols-1 md:grid-cols-2">
           {/** first row: Name */}
-          <NameForm name={multisig.name} editable={isSigner} teamId={multisig.id} />
+          <div className="flex gap-4 justify-between">
+            <NameForm name={multisig.name} editable={isSigner} teamId={multisig.id} />
+            <DescriptionForm
+              description={multisig.description || 'Add a Multisig description'}
```

Comment:
```suggestion
              description={multisig.description || ''}
```

I'd prefer not giving it a prefilled "Add a Multisig description"

"Add a Multisig description" should appear as a placeholder (e.g. with some opacity)

Reply:
Over here for example, "Add a Multisig description" should be grayed out when there's no description

![Screenshot 2024-05-14 at 12 49 00](https://github.com/TalismanSociety/signet-web/assets/81092286/46c7dd83-5c43-4cfe-9537-09f067467b3b)